### DISCUSSION TITLE
Features for shared accounts

### DISCUSSION THREAD

**Discussion by makenova** [actualbudget/releases#23] [Feature ideas]
I share an account with my partner and I'm only responsible for a portion of it.
Actual does not currently have an obvious way to way to handle such accounts.

The simplest way I imagine to handle a shared account would be to:

Split a transaction and assign some of the transaction to off-budget.
When the balance it paid off, the portion you are not responsible for would also need to be set to off-budget and match the previous off-budget sub-transactions.

This seems to be the most straight forward and low impact approach but it's likely that I'm missing something.
If you can think of another work around with the current features available or another feature to enable shared accounts, please share below.

**Comment by teakdecks** [comment]
I recently was using nYNAB with a shared account with a housemate, this is how I managed it. It might help, or it might not actually match how you and your partner have things set up. Note- this exact workflow is clunky depending on the types of transactions you have. We used the joint account for paying shared bills, and doing occasional food shops/days out together, and both paid in equal amounts

I had the entire account on budget, and a budget line of "joint account". Any money in the joint account budget line was my housemate's money. This category is effectively 'off budget'- in the sense of transferring out of it would mean I was stealing from my mate. Any money he put in each month went directly into the category, to avoid calling it my income. Whenever a transaction came out, I split it 50/50 between that category and whatever actual category it was. This was fine because 75% of the transactions were regular, scheduled transactions, so I just had to set them up once and then approve them. It was worth putting up with the faff of putting in the odd groceries shop for the benefit of easy reconciliation. 

Note- with the current scheduled transactions in Actual you couldn't do this. 

On the flip side, if you and your partner split things in the sense of you pay for electrics and gas, and they pay for internet and council tax, then it's pretty simple. Same as above, but you don't have to split each transaction, just assign the appropriate category for it.

I'm not sure if this provides an admittedly slightly clunky solution to your problem or not? The main difference between your description and mine (as far as I can understand) is that mine keeps it all in one account rather than making an on and an off budget account.
